### DISCUSSION TITLE
PC1555MX Issues (No Read/Data)

### DISCUSSION THREAD

**Discussion by chrisb070819** [taligentx/dscKeybusInterface#137] [dscKeybusInterface Discussion]
I will be first to admit that this small electronics section is pretty new to me. The security system is a PC1555MX that has 6 zone and all in use. The system also has 2 control pads that are the PC1555RKZ. I am planning to hook this up to a ESP8266 NodeMCUv2. I am powering the NodeMCU via the micro USB. I have tried using the KeybusReader and the VirtualKeypad-Web. VirtualKeypad-Web is what I would like to use in the end. I have the wiring I setup based on documentation and placed below on what I read. 
                          
CLK(Yellow) > 15K  > NodeMCU D1
                      > 10K > Ground
Data (Green) > NPN Collector (2N3904)
                       > 15K > NodeMCU D2
                                  > 10K > G
NodeMCU D8 > 1K > NPN Base (2N3904)
NPN Emitter (2N3904) > Ground

I have tested my resistance down the connection and got 15k from CLK start to NodeMCU D1 In. I also got the 25K from CLK start to Ground.
I did the same for Data. From Data start to NodeMCU D2 in I got 15K and Data start to Ground was 25K.    

I was reading a little on how some have adjusted the resistors to fit their setup but i'm not quite sure what I should be getting. Additionally I am not sure if having 2 control pads impacts this setup. I have also tried using a breadboard instead of my soldering and got the same results. I have tried quite a few things and not really sure where I am going wrong. 

Any advice for being able to narrow down my issue? Test points, expected voltages, perhaps code things.

**Comment by taligentx** [comment]
Hi @chrisb070819 - late response, but what output do you get from the `KeybusReader` sketch?  Also, if using USB to power the NodeMCU, make sure the grounds are connected from the DSC panel (black wire) to the NodeMCU.

The number of keypads shouldn't affect the interface, it operates like any other accessory on the DSC Keybus.
